which => these (#1276)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/ApiOverview/SiteHandling/Basics.rst

======

.. note::
   Site handling is available since TYPO3 9 LTS.

TYPO3 site handling and configuration is the starting point for creating new web sites. 

> The corresponding modules are found in the TYPO3 backend in the section "Site management".

Reviewer: ```suggestion
The corresponding modules are found in the TYPO3 backend in the section :guilabel:`Site management`.
```

in the section "Site management".
TYPO3 site handling and configuration is the starting point for creating new web sites. 
The corresponding modules are found in the TYPO3 backend in the section "Site management".

A site configuration consists of the following parts:

> * Base URL configuration: Under which domain(s) is my site accessible

Reviewer: This actually was a question with question mark thus actually OK.

Author: The answer stands before. So it cannot be a question any more.

> * Base URL configurations: Under these domain(s) my site is accessible.

Reviewer: ```suggestion
* Base URL configurations: the domain(s) to access my site:
```

> * :ref:`Language configuration<sitehandling-addingLanguages>`: These languages are available for my site.

Reviewer: ```suggestion
* :ref:`Language configuration<sitehandling-addingLanguages>`: the languages of my site.
```

> * :ref:`Error Handling<sitehandling-errorHandling>`: How should errors for this site behave (For example: configuration of custom 404 pages)

Reviewer: ```suggestion
* :ref:`Error Handling<sitehandling-errorHandling>`: error behavior of my site (For example: configuration of custom 404 pages)
```

Author: Which English is to be used for TYPO3?

Reviewer: A good question but right now there is no strict rule here. Personally I'd try to aim at American English though.

> * :ref:`Static Routes<sitehandling-staticRoutes>`: Add static routes to a site (For example :file:`robots.txt` on a per site base)

Reviewer: ```suggestion
* :ref:`Static Routes<sitehandling-staticRoutes>`: static routes of my site (For example :file:`robots.txt` on a per site base)
```
